stack.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+
+class Element(object):
+    ''' Create data element with default value and previous pointer. '''
+    def __init__(self, value=None, previous=None):
+        ''' Value and previous pointer default to none. '''
+        self.val = value
+        self.previous = previous
```

Comment:
Is there ever any reason to create an element with no value?  If not, why make it an optional argument.